samples/node/createJob.js

```diff
@@ -0,0 +1,61 @@
+const credentials = require('./credentials').credentials;
+const SkySyncClient = require('skysync-cli').SkySyncClient;
+
+const client = new SkySyncClient(credentials);
+
+var nfsConnection;
+var o365Connection;
+
+var nfsConnectionPromise = client.connections.add({
+	name: 'Sample NFS Connection',
+	platform: { id: 'nfs' },
+	auth: {
+		uri: '//test-server',
+		username: 'localhost/admin',
+		password: 'secret'
+	}
+}).then(connection => { nfsConnection = connection });
```

Comment:
So I'm wondering why we don't use a script like [this](https://github.com/portalarchitects/usfs/blob/master/scripts/create-connections.js) where for example you'd create connections with the skysync-cli command like

```javascript
function createConnection(platform, name, authFile) {
	let cmd = `skysync-cli connections add --platform ${platform } --name ${name} --auth-file ${authFile}`;
	console.log(cmd);
	return exec(cmd);
}
```

Reply:
This is showing how you could create jobs using the SDK rather than the CLI.